### PR TITLE
[Feature] add onboarding question from ATS hire

### LINKED ISSUE #625: [Feature] add onboarding question from ATS hire
[Idea] Is there a way we could get more employee info from app process, so that more fields are auto filled going into onboarding?

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #625, #760
close with associatedemployers/quartz#102
close with associatedemployers/igneous#310

onboarding

### CODE REVIEW

**Review by James1x0 — APPROVED** [review]

**Comment by JavaLavaMT** [comment]
close with associatedemployers/igneous#355

File: app/controllers/account/job-opening/campaign/applicant-tracking.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export default class AccountJobOpeningCampaignApplicantTrackingController extend
   confirmDisqualifyModalId = 'modal__ats-confirm-disqualify'
   disqualifyModalId =        'modal__ats-disqualify'
   schedulerModalId =         'modal__ats-scheduler'
+  confirmHireModalId=        'modal__ats-confirm-hire'
```
Comment (James1x0) [inline app/controllers/account/job-opening/campaign/applicant-tracking.js:34]:
```suggestion
  confirmHireModalId =       'modal__ats-confirm-hire'
```

File: app/models/employee.js [hunk 2]
```diff
@@ -61,6 +61,16 @@ export default class Employee extends Model.extend(Validations) {
...
 
+  @attr('Boolean') autoOnboarding;
+
+  @attr('date') delayOnboardingResponseDate;
+
+  @belongsTo('form', {
+    async:   true,
+    inverse: null
+  })
+  OnboardingQuestionForm;
```
Comment (James1x0) [inline app/models/employee.js:72]:
```suggestion
  onboardingQuestionForm;
```
That's a bug

File: app/styles/granitestyles/components/modals/ats-hire.less [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.modal__ats-employee-hire-confirm{
+  width:24em !important;
+}
```
Comment (James1x0) [inline app/styles/granitestyles/components/modals/ats-hire.less:3]: We need to do something else here. A static width will break mobile.

File: app/components/modals/ats-hire.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import Component from '@ember/component';
+import $ from 'jquery';
+import { action } from '@ember/object';
+
+export default class AtsHireComponent extends Component {
+  employeeFields= [
```
Comment (James1x0) [inline app/components/modals/ats-hire.js:6]:
```suggestion
  employeeFields = [
```

**Review by James1x0 — CHANGES_REQUESTED** [review]
One last change. I think for the modal, you can increase the overall size by adding the `large` class to the modal.

**Comment by James1x0** [comment]
@JavaLavaMT Once the width thing is fixed, please go to the review app: https://granite-prod-pr-762.herokuapp.com/

And just QA what you've added. I'm interested in making sure you check out the changes made to `form-element` and make sure those are a-ok.

**Pull request merged**